[2024-11-20T23:05:21.752+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-20T23:05:21.763+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_heart_disease_data.split_data manual__2024-11-20T23:05:19.178036+00:00 [queued]>
[2024-11-20T23:05:21.768+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_heart_disease_data.split_data manual__2024-11-20T23:05:19.178036+00:00 [queued]>
[2024-11-20T23:05:21.768+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2024-11-20T23:05:21.779+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): split_data> on 2024-11-20 23:05:19.178036+00:00
[2024-11-20T23:05:21.788+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=16178) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-20T23:05:21.787+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'process_heart_disease_data', 'split_data', 'manual__2024-11-20T23:05:19.178036+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp8krr1jht']
[2024-11-20T23:05:21.789+0000] {standard_task_runner.py:72} INFO - Started process 16191 to run task
[2024-11-20T23:05:21.789+0000] {standard_task_runner.py:105} INFO - Job 44: Subtask split_data
[2024-11-20T23:05:21.823+0000] {task_command.py:467} INFO - Running <TaskInstance: process_heart_disease_data.split_data manual__2024-11-20T23:05:19.178036+00:00 [running]> on host 4c4aee67b64f
[2024-11-20T23:05:21.875+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_heart_disease_data' AIRFLOW_CTX_TASK_ID='split_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-20T23:05:19.178036+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-20T23:05:19.178036+00:00'
[2024-11-20T23:05:21.876+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-20T23:05:21.928+0000] {logging_mixin.py:190} INFO - Dane podzielone na 'modelowy.csv' i 'douczeniowy.csv'
[2024-11-20T23:05:21.929+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-20T23:05:21.935+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-20T23:05:21.936+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=process_heart_disease_data, task_id=split_data, run_id=manual__2024-11-20T23:05:19.178036+00:00, execution_date=20241120T230519, start_date=20241120T230521, end_date=20241120T230521
[2024-11-20T23:05:21.963+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-20T23:05:21.977+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-20T23:05:21.978+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
